lua/neoterm.lua

```diff
@@ -154,33 +154,35 @@ function neoterm.open(opts)
       border = "single",
     }
 
+    local width_padding = 2 -- Add some padding so that terminal fits within the visible area (neovide)
```

Comment:
Given this seems to be a neovide specific issue, I think we can only do this for neovide users.
```suggestion
    -- Fix for neovide where the terminal ends up outside of the visible area
    local width_padding = vim.g.neovide and 2 or 0
```

Reply:
fd70ef2cbca52148e7af38b005602ecdda7f539b will work with this great suggestion!